lib/jekyll/polyglot/patches/jekyll/site.rb

```diff
@@ -108,19 +108,23 @@ def process_active_language
       @exclude = old_exclude
     end
 
+    def split_on_multiple_delimiters(string)
+      delimiters = ['.', '/']
```

Comment:
Do you think we should include more delimiters, or maybe even add this as an option in `_config.yml` and set some default values? Because maybe the user wants to use a different delimiter, for instance `-` or `_`, but we avoid using this by default since someone might end up using it in the file name. E.g. a blog post like `traduzindo_para_en_us` even though the post is in `pt-br` it has `en_us` in the name.

Reply:
I could be amenable to adding `_` as an additional separator, but not `-` as that interacts poorly with the hyphenated `pt-br` and `zh-cn` language codes.

I don't want to make the seperator options more configurable however, as the `derive_lang_from_path` still only works when `lang_from_path: true` is set in the config. Excessive configuration is not a feature, and can lead to bugs.

Reply:
It makes sense.

Reply:
Should we also update my code to support splitting by these chars?

Reply:
you can submit another PR with updates to support `_` or other separators, if you can also update the tests. Theres a great example now, so it should be doable. These are the only filepath separators I want to support with this release. This is still a big improvement tho.